### PR TITLE
Blog post on the Cake for Rider release 0.3.0

### PR DESCRIPTION
**Pull request by nils-a**

### CODE REVIEW

File: input/blog/2021-04-01-cake-rider-0.3.0-released.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Cake for Rider 0.3.0 released
+category: Release Notes
+author: nils-a
+---
+
+Version 0.3.0 of [Cake for Rider](https://plugins.jetbrains.com/plugin/15729-cake-rider) has been released.
+
+This release features, among some small changes:
```
Comment (augustoproiete) [inline input/blog/2021-04-01-cake-rider-0.3.0-released.md:9]: Small suggestion:

```suggestion
Highlights of this release include:
```

**Review by augustoproiete — APPROVED** [review]
LGTM

File: input/blog/2021-04-01-cake-rider-0.3.0-released.md (hunk 1 above)
Comment (nils-a) [reply input/blog/2021-04-01-cake-rider-0.3.0-released.md]: 👍

**Comment by nils-a** [comment]
[Cake for Rider 0.3.0](https://github.com/cake-build/cake-rider/releases/tag/0.3.0) has been submitted and is now in moderation. I will merge this PR once the package habe been approved by JetBrains.

**Pull request merged**